### PR TITLE
Button to set LSQT to true/false (on view page)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: public/bitemp_graph.js [hunk 1]
```diff
@@ -526,7 +526,7 @@ var barChart = function() {
         })
         .text(function(d) {
           var str = '';
-          if(window.location.href.endsWith('/search')) {
+          if(window.location.href.endsWith('/')) {
```
Comment (dmcassel) [inline public/bitemp_graph.js]: I think "window.location.pathname === '/'" would be a better test here. That avoids the possibility of someone typing an extra slash and future-proofs against using URL parameters at some point (which are included in .href but not in .pathname).

File: public/bitemp_graph_helper.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ var addTempColls = function(id, search) {
       //sorts alphabetically
       dropArray.sort();
 
-      //Append the collection names to the drop down list
+      //Append the u names to the drop down list
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: Was this on purpose? If so, it's not clear what it refers to.

File: public/bitemp_graph_helper.js [hunk 3]
```diff
@@ -644,6 +645,18 @@ var getBarChart = function (params, docProp) {
     getBarChart(params, selectedText);
   });
 
+  $('#myonoffswitch').change( function() {
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: Can we get a better name for this? What is it switching on and off?

File: public/bitemp_graph_helper.js (hunk 2 above)
Comment (kmcostel) [inline public/bitemp_graph_helper.js]: LOL Ash you missed one.

File: public/bitemp_graph_helper.js (hunk 3 above)
Comment (kmcostel) [inline public/bitemp_graph_helper.js]: Ash... it's turning the lsqt on and off, perhaps lsqtSwitch?

**Pull request merged**